karma.conf.js

```diff
@@ -98,7 +98,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Firefox'],
+    browsers: ['Chrome'],
```

Comment:
This will cause a problem for Coveralls, won't it?

Reply:
@mxsnyder yea crap, sorry forgot to roll this back when i was debugging.